components/input/ResizableTextArea.tsx

```diff
@@ -75,7 +86,7 @@ class ResizableTextArea extends React.Component<TextAreaProps, TextAreaState> {
   }
 
   renderTextArea = () => {
-    const { prefixCls, autoSize, autosize, className, disabled } = this.props;
```

Comment:
Hi, could you please explain why we need to remove this props?

Reply:
Because `disabled` will cause `onResize` not to trigger.